packages/core/src/app/shipping/StaticConsignment.tsx

```diff
@@ -48,7 +48,7 @@ const StaticConsignment: FunctionComponent<StaticConsignmentProps> = ({
                         <strong>
                             <TranslatedString id="shipping.shipping_method_label" />
                         </strong> }
-                    <div className="shippingOption shippingOption--alt">
+                    <div className="shippingOption shippingOption--alt shippingOption--state-selected">
```

Comment:
This class name gives the impression that `state` is selected, can we just call it `shippingOption--selected`? Can't think of a better name at the moment :|

Reply:
Renamed.